src/test/java/org/gridsuite/loadflow/server/LoadFlowParametersTest.java

```diff
@@ -0,0 +1,207 @@
+        String parametersToCreateJson = mapper.writeValueAsString(parametersToCreate);
+
+        mockMvc.perform(post(URI_PARAMETERS_BASE).content(parametersToCreateJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk()).andReturn();
+
+        LoadFlowParametersInfos createdParameters = parametersRepository.findAll().get(0).toLoadFlowParametersInfos();
+
+        assertThat(createdParameters).recursivelyEquals(parametersToCreate);
+    }
+
+    @Test
+    public void testCreateWithDefaultValues() throws Exception {
+        LoadFlowParametersInfos defaultParameters = LoadFlowParametersInfos.builder()
+            .commonParameters(LoadFlowParameters.load())
+            .specificParametersPerProvider(Map.of())
+            .build();
+
+        mockMvc.perform(post(URI_PARAMETERS_BASE + "/default"))
+                .andExpect(status().isOk()).andReturn();
+
+        LoadFlowParametersInfos createdParameters = parametersRepository.findAll().get(0).toLoadFlowParametersInfos();
+
+        assertThat(createdParameters).recursivelyEquals(defaultParameters);
+    }
+
+    @Test
+    public void testRead() throws Exception {
+
+        LoadFlowParametersInfos parametersToRead = buildParameters();
+
+        UUID parametersUuid = saveAndRetunId(parametersToRead);
+
+        MvcResult mvcResult = mockMvc.perform(get(URI_PARAMETERS_GET_PUT + parametersUuid))
+                .andExpect(status().isOk()).andReturn();
+        String resultAsString = mvcResult.getResponse().getContentAsString();
+        LoadFlowParametersInfos receivedParameters = mapper.readValue(resultAsString, new TypeReference<>() {
+        });
+
+        assertThat(receivedParameters).recursivelyEquals(parametersToRead);
+    }
+
+    @Test
+    public void testUpdate() throws Exception {
+
+        LoadFlowParametersInfos parametersToUpdate = buildParameters();
+
+        UUID parametersUuid = saveAndRetunId(parametersToUpdate);
+
+        parametersToUpdate = buildParametersUpdate();
+
+        String parametersToUpdateJson = mapper.writeValueAsString(parametersToUpdate);
+
+        mockMvc.perform(put(URI_PARAMETERS_GET_PUT + parametersUuid).content(parametersToUpdateJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+
+        LoadFlowParametersInfos updatedParameters = parametersRepository.findById(parametersUuid).get().toLoadFlowParametersInfos();
+
+        assertThat(updatedParameters).recursivelyEquals(parametersToUpdate);
+    }
+
+    @Test
+    public void testDelete() throws Exception {
+
+        LoadFlowParametersInfos parametersToDelete = buildParameters();
+
+        UUID parametersUuid = saveAndRetunId(parametersToDelete);
+
+        mockMvc.perform(delete(URI_PARAMETERS_GET_PUT + parametersUuid)).andExpect(status().isOk()).andReturn();
+
+        List<LoadFlowParametersEntity> storedParameters = parametersRepository.findAll();
+
+        assertTrue(storedParameters.isEmpty());
```

Comment:
be consistent and use `assertThat()` everywhere
```suggestion
        assertThat(storedParameters).isEmpty();
```